src/components/MediaSettings/VideoBackgroundEditor.vue

```diff
@@ -54,25 +60,43 @@
 				:size="40"
 				fill-color="#fff" />
 		</button>
+		<!--native file picker, hidden -->
+		<input v-show="false"
+			id="file-upload"
+			ref="fileUploadInput"
+			multiple
+			type="file"
+			tabindex="-1"
+			aria-hidden="true"
+			@change="handleFileInput">
```

Comment:
**issue (blocking):** duplicated id

We have `id="file-upload"` already twice in the app. And they may appear simultaneously.

Maybe call it `custom-background-file` or anything else more specific?